packages/hawtio/src/plugins/shared/__mocks__/connect-service.ts

```diff
@@ -72,7 +76,7 @@ class MockConnectService implements IConnectService {
     return ''
   }
 
-  getJolokiaUrlFromName(name: string): string | null {
+  getJolokiaUrlFromId(name: string): string | null {
```

Comment:
Thanks - I should've done it when adding ID support for connections...

Reply:
👍